app/components/chat_form/chat_form.css

```diff
@@ -4,6 +4,19 @@
   align-items: flex-end;
 }
 
+.ChatForm h3 {
+  width: 100%;
+  padding: var(--spacing-unit-s);
```

Comment:
```suggestion
  padding: var(--spacing-unit-s) 0;
```

To me this looks better left-aligned witht the textarea.

Before:
![Screenshot from 2024-06-05 12-19-12](https://github.com/tactilenews/100eyes/assets/26943915/e6fb254e-65df-49f2-88de-31986daf6a88)


After:
![Screenshot from 2024-06-05 12-18-51](https://github.com/tactilenews/100eyes/assets/26943915/0bce8732-09df-4fc0-b6de-fa9c1bd6725e)